src/main/scala/tech/cryptonomic/conseil/tezos/ApiNetworkOperations.scala

```diff
@@ -0,0 +1,42 @@
+package tech.cryptonomic.conseil.tezos
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directive1
+import akka.http.scaladsl.server.Directives.{complete, provide}
+import com.typesafe.config.{Config, ConfigFactory}
+import slick.jdbc.PostgresProfile.api._
+import tech.cryptonomic.conseil.db.DatabaseApiFiltering
+
+import scala.concurrent.ExecutionContext
+import scala.util.Try
+
+
+object ApiNetworkOperations {
+  private lazy val networkOperationsMap: Map[String, Database] = getDatabaseMap
+
+  def apply(): ApiNetworkOperations = new ApiNetworkOperations(networkOperationsMap)
+
+  def getDatabaseMap: Map[String, Database] = {
+    val config: Config = ConfigFactory.load()
+    NetworkConfigOperations.getNetworks(config).map { network =>
```

Comment:
Don't forget to check both the network and platform everywhere!

Reply:
changed